modules/setting/setting.go

```diff
@@ -480,6 +480,16 @@ func NewContext() {
 	}[Cfg.Section("time").Key("FORMAT").MustString("RFC1123")]
 
 	RunUser = Cfg.Section("").Key("RUN_USER").String()
+
+	switch mode := Cfg.Section("").Key("RUN_MODE").String(); mode {
+	case "prod":
+		ProdMode = true
+	case "dev":
+		ProdMode = false
+	case "test":
```

Comment:
Absolutely, nothing. I just follow the standards, which consists of `prod`, `dev` and `test`.